packages/theme/src/theme/_buttons.scss

```diff
@@ -74,27 +88,12 @@
 	&-link {
 		font-size: $font-size-base;
 		font-weight: inherit;
-
-		&.btn-icon-only {
-			color: $gray;
-
-			&:focus,
-			&:hover {
-				color: $scooter;
-			}
-
-			&:active {
-				color: darken($scooter, 12%);
-			}
-		}
+		@include svg-button-behavior();
 	}
 
 	&-icon-only {
 		display: inline-block;
-
-		svg {
-			vertical-align: middle;
-		}
+		@include svg-button-behavior();
```

Comment:
Finally, only add mouse behaviors on `btn-icon-only` and `btn-link`.
Can't add the behavior globally on svg because I don't need any behavior on _only_ icons, see http://2124.talend.surge.sh/components/?selectedKind=Enumeration&selectedStory=with%20icon&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel